pipelines/smartseq2_single_sample/GroupQCOutputs.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 21 11:07:46 2018
+
+@author: jishuxu
+"""
+
+from crimson import picard
+import argparse
+import os
+import pandas as pd
+
+def AggregatePicardMetricsRow(filenames, output_name):
+    """
+    pipeline output picard QC metrics at sinle cell/sample level.
+    This functin is called to merge/aggregate QC metrics by metrics type and
+    then merge multiple QC measurement into single matrix file.
+    In this file, column is sample/cell and row is QC metrics
+    :param files: metric files from pipeline outputs
+    :param met_name: metrics name with workflow name and subworkflow name
+    as prefix.such as 'run_pipelines.RunStarPipeline.alignment_summary_metrics'
+    """
+    # initial output
+    mets = {}
```

Comment:
Tiny nit: I prefer avoiding abbreviations in variable names when possible, e.g using `metrics` instead of `mets` for clarity